packages/flutter_tools/lib/src/base/common.dart

```diff
@@ -4,3 +4,8 @@
 
 const int observatoryDefaultPort = 8181;
 const int diagnosticDefaultPort = 8182;
+
+// Names of some of the Timeline events we care about
+const String flutterEngineMainEnterEventName = 'FlutterEngineMainEnter';
```

Comment:
Yeah, but `lib/src/service_protocol.dart` depends on `device.dart` and I'd like to avoid unnecessary cyclic dependencies.